### DISCUSSION TITLE
Welcome to open-source-flash Discussions!

### DISCUSSION THREAD

**Discussion by pakastin** [open-source-flash/open-source-flash#137] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Brian151** [comment]
I'm Brian, hacker/programmer and game asset ripper

I mostly just passively follow the latest updates here, however I work off and on with the director/shockwave formats and I plan to use a certain portion of SWF for future projects as I believe the usefulness of the format is greatly underplayed for the sake of "screw flash"/"screw adobe". I also am working on porting a few flash or shockwave games, progress is slow. I also am a member of the Crystalien Redux project which, exclusively focuses on modding a single flash adver-game from 2007-2008.

**Reply by pakastin** [reply]
Cool! Sounds very interesting! 👌

**Comment by Brian151** [comment]
and seems i cannot reply to your reply with pale moon @pakastin... 

i actually have a lot planned in terms of SWF already. it's just i'm lazy/unmotivated, partly because parts of the existing codec aren't fun to deal with (variable length bitfields, but i do have a decoder, at least). my irl is also [intentionally] kept extremely busy. when i'm actually home, i'm too tired to even play games or watch youtube anymore. the last few weeks also have been a royal pain as my dad's health issues have been acting-up... anyways, hopefully i'll have something soon™
